crates/router/src/connector/worldline/transformers.rs

```diff
@@ -361,6 +390,15 @@ impl<F, T> TryFrom<types::ResponseRouterData<F, PaymentResponse, T, types::Payme
         })
     }
 }
+#[derive(Default, Debug, Serialize)]
+pub struct ApproveRequest {}
+
+impl TryFrom<&types::PaymentsCaptureData> for ApproveRequest {
+    type Error = error_stack::Report<errors::ParsingError>;
+    fn try_from(_item: &types::PaymentsCaptureData) -> Result<Self, Self::Error> {
+        Ok(Self {})
+    }
+}
```

Comment:
```suggestion
#[derive(Default, Debug, Serialize)]
pub struct ApproveRequest;

impl TryFrom<&types::PaymentsCaptureData> for ApproveRequest {
    type Error = error_stack::Report<errors::ParsingError>;
    fn try_from(_item: &types::PaymentsCaptureData) -> Result<Self, Self::Error> {
        Ok(Self)
    }
}
```

Reply:
ApproveRequest is a request body. If we accommodate the suggested changes this will send empty body in the http request which connector doesn't support. As discussed resolving this comment.